pipelines/h5c/rtp/v1/stage_1_task_scripts/do_LIBRARIAN_XRFI.sh

```diff
@@ -18,21 +18,30 @@ if [ "${upload_to_librarian}" == "True" ]; then
     if [ "${librarian_xrfi}" == "True" ]; then
         # get the integer portion of the JD
         jd=$(get_int_jd ${fn})
+        decimal_jd=$(get_jd ${fn})
 
         # get xrfi folder
         xrfi_folder=`echo ${fn%.sum.uvh5}.xrfi`
         xrfi_stage_1_folder=`echo ${fn%.sum.uvh5}.stage_1_xrfi`
         compressed_file=`echo ${fn%.sum.uvh5}.stage_1_xrfi.tar.gz`
         if [ -d "${xrfi_folder}" ]; then
```

Comment:
I am not sure whether there are any scenarios that, within one jd, there are file names mixed with .xrfi and .stage_1_xrfi.
I guess not, then this code should work. Please let me know if it is not the case. Thanks!

Reply:
Since this script runs after `do_XRFI_DAY_THRESHOLD.sh`, I believe everything will have been renamed. SO we don't need this if-else.

(By the way, the renaming was done to prevent conflicts with stage 2 analysis. With the zipping, that may be less important but I want to keep it for consistency.)

Reply:
Thanks for the feedback!
This is now done in https://github.com/HERA-Team/hera_pipelines/pull/16/commits/394def6a2e5bc5d0a68d537f31eb0730eefd6d13